src/data-workspace/category-combo-table-body/category-combo-table-body-header.js

```diff
@@ -55,6 +55,7 @@ export const CategoryComboTableBodyHeader = ({
     categoryOptionCombos,
     categories,
     checkTableActive,
+    allNonNumberValueType,
```

Comment:
Minor, but I don't think this is very descriptive as to what it does? Maybe it could just be called `showTotalHeader` or something similiar, that describes the functionality?

Reply:
I've updated to very dry `hideColumnTotalsDueToNonNumberValueType` and `hideRowTotalsDueToNonNumberValueTypes`